package.json

```diff
@@ -1,23 +1,38 @@
 {
-  "name": "Attendize",
-  "version": "2.0.0",
+  "name": "attendize",
+  "version": "1.0.0",
   "description": "Attendize Ticketing Platform",
   "main": "public/index.php",
   "dependencies": {
-    "less": "^3.9.0"
+    "less": "^3.10.3"
   },
+  "private": true,
   "devDependencies": {
+    "axios": "^0.19",
     "bower": "^1.8.8",
-    "grunt": "^1.0.3",
+    "cross-env": "^6.0",
+    "grunt": "^1.0.4",
     "grunt-cli": "^1.3.2",
     "grunt-contrib-concat": "^1.0.1",
     "grunt-contrib-less": "^2.0.0",
-    "grunt-contrib-uglify": "^4.0.0",
+    "grunt-contrib-uglify": "^4.0.1",
     "grunt-contrib-watch": "^1.1.0",
-    "laravel-elixir": "6.0.0-18"
+    "laravel-elixir": "6.0.0-18",
+    "laravel-mix": "^5.0.0",
+    "lodash": "^4.17.15",
+    "resolve-url-loader": "^3.1.1",
+    "sass": "^1.23.7",
+    "sass-loader": "^8.0.0"
   },
   "scripts": {
     "build-frontend": "grunt",
+    "dev": "npm run development",
+    "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "watch": "npm run development -- --watch",
+    "watch-poll": "npm run watch -- --watch-poll",
+    "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "prod": "npm run production",
+    "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
```

Comment:
Like I mentioned with the laravel mix file removal, please can you also remove this in favor of us making a new PR to move all the frontend dependencies out of bower and into yarn, as well as move away from grunt and migrate over to laravel mix. We'd like to do that as a separate task and it's outside of the scope of Upgrading laravel here.